research-software-directory/RSD-as-a-service
# 192 project page

# Project page - mentions section (impact & output)

Closes #192
Closes #193 
Closes #194  

Changes proposed in this pull request:
* Users can see all mentions for selected project. The section contains two parts: impact and output. Both parts support "featured" items: the items that are show at the top showing the image, author, date and link to website.
* Users can see team members, related projects and tools
* Refactoring of software page to share components with project page
* Fixes for organisation page and contact person email
* Adding initial test for project page

How to test:

* `docker-compose down --volumes`: remove old data just to be sure ;-)
* `docker-compose build`: build the solution
* `docker-compose up`: start the solution
* perform data migration
* Visit following projects to experience variations in the mentions section:
  * eStep: has featured output
  * 3d-e-chem: has impact item
  * AA-alert: has only output
  * a-light-in-the-dark: has no mentions (no section is shown)
* View more projects and navigate between project, software and organisation pages using, participating organisation, related projects and tools links.

Example 3d-e-chem page
![image](https://user-images.githubusercontent.com/9204081/162580483-ceab1529-e761-4b14-a4b0-f61d84341d60.png)

PR Checklist:

*   [n/a] Update documentation
